servers/src/main/java/tachyon/worker/block/TieredBlockStore.java

```diff
@@ -85,7 +85,8 @@
 
   public TieredBlockStore(TachyonConf tachyonConf) throws IOException {
     mTachyonConf = Preconditions.checkNotNull(tachyonConf);
-    mMetaManager = BlockMetadataManager.newBlockMetadataManager(mTachyonConf);
+    mMetaManager =
+        Preconditions.checkNotNull(BlockMetadataManager.newBlockMetadataManager(mTachyonConf));
```

Comment:
make sense, you mean move the check to here, right? 
https://github.com/amplab/tachyon/blob/master/servers/src/main/java/tachyon/worker/block/allocator/AllocatorFactory.java#L32
https://github.com/amplab/tachyon/blob/master/servers/src/main/java/tachyon/worker/block/evictor/EvictorFactory.java#L33